integration/actions/namespace_test.go

```diff
@@ -139,10 +137,3 @@ func executeDeleteNamespaceAction(namespace string) error {
 	log.Printf("deleting namespace output: \n%s\n", string(o))
 	return nil
 }
-
-func getTestNamespace() string {
-	tName := fmt.Sprintf("TestAction-%s", runtime.GOOS)
-	name := strings.ToLower(fmt.Sprintf("%s-%d", tName, time.Now().Unix()))
```

Comment:
I am curious how we have been able to generate an identical name if we are composing it with `time.Now().Unix()`

Reply:
maybe it was being trimmed somewhere?